.github/workflows/ci.yml

```diff
@@ -188,6 +187,7 @@ jobs:
         timeout-minutes: 10
         with:
           cache: restore
+          java-version: '23'
```

Comment:
24 ?

Reply:
No, this will break due to the incubator module usage. This will come when we switch to hard 24 requirement.